packages/utils/src/route.js

```diff
@@ -177,6 +177,9 @@ export const createRoutes = function createRoutes ({
     if (trailingSlash !== undefined) {
       route.pathToRegexpOptions = { ...route.pathToRegexpOptions, strict: true }
       route.path = route.path.replace(/\/+$/, '') + (trailingSlash ? '/' : '')
+      if (route.path === '') {
```

Comment:
how about?
`route.path = route.path.replace(/\/+$/, '') + (trailingSlash ? '/' : '') || '/'`

Reply:
looks good.
I'll adopt this way, thx!